src/applications/static-pages/authenticated-homepage/components/App/App.js

```diff
@@ -0,0 +1,53 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { selectShowAuthenticatedHomepage } from 'applications/static-pages/authenticated-homepage/selectors';
+import environment from 'platform/utilities/environment';
+
+export const App = ({ showAuthHomePage }) => {
+  const homepage = document.getElementById('homepage');
```

Comment:
Piggybacking on Kelson's earlier point, would it make sense to throw an Error if it can't find this element ID, to serve as an indicator that that ID name is verrrrrry important?